boot/zephyr/main.c

```diff
@@ -73,7 +81,7 @@ void main(void)
 	SYS_LOG_INF("Setting vector table to %p", vt);
 
 	/* Not all targets set the VTOR, so just set it. */
-	_scs_relocate_vector_table((void *) vt);
+	relocate_vector_table((void *) vt);
```

Comment:
For reference: https://gerrit.zephyrproject.org/r/gitweb?p=zephyr.git;a=blob;f=include/flash.h;h=1d90a22fd2f0e47bcecb938f06b73ac73f794d57;hb=refs/heads/master#l107